packages/frontend/src/components/CreateUserForm/VerifyEmailForm.tsx

```diff
@@ -0,0 +1,75 @@
+import { Intent } from '@blueprintjs/core';
+import React, { FC, useState } from 'react';
+import Countdown, { zeroPad } from 'react-countdown';
+import { useForm } from 'react-hook-form';
+import {
+    AnchorLinkWrapper,
+    ErrorMessage,
+    Subtitle,
+} from '../../pages/SignUp.styles';
+import AnchorLink from '../common/AnchorLink';
+import Form from '../ReactHookForm/Form';
+import { PasswordInputField, SubmitButton } from './CreateUserForm.styles';
+
+export const VerifyEmailForm: FC<{
+    numberOfAttempts: number;
+    expirationTime: string;
+    onSubmit: (code: string) => void;
+    onResend: () => void;
+}> = ({ numberOfAttempts, expirationTime, onSubmit, onResend }) => {
+    const methods = useForm({
+        mode: 'onSubmit',
+        defaultValues: {
+            code: '',
+        },
+    });
+    return (
+        <Form name="verify-email" onSubmit={onSubmit} methods={methods}>
```

Comment:
the form onSubmit is going to return an object with all the form values. `{ code: string }` and our callback function is expecting `(code:string)`